dapps/assets/src/components/digitaltwins/DigitalTwins.ts

```diff
@@ -125,4 +161,42 @@ export default class DigitalTwinsComponent extends mixins(EvanComponent) {
       this.isActiveSearch = false;
     }
   }
+
+  async addFavorite(twin) {
```

Comment:
using Set() would make it much easier to implement add/remove functions. https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Set